.circleci/config.yml

```
@@ -68,10 +68,10 @@ jobs:
             - *cache_key
 
       # Build and Test
-      - write_master_hash
+      - write_main_hash
       - run:
           name: Run Affected Lint Checks
-          command: yarn nx affected:lint --base=$(cat ~/project/master.txt) --head=$CIRCLE_SHA1 --parallel=1
+          command: yarn nx affected:lint --base=$(cat ~/project/main.txt) --head=$CIRCLE_SHA1 --parallel=1
```

These can be removed

```suggestion
          command: yarn nx affected:lint  --head=$CIRCLE_SHA1 --parallel=1
```